addons/l10n_sa_edi/models/res_partner.py

```diff
@@ -0,0 +1,28 @@
+from odoo import fields, models, api
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    l10n_sa_edi_building_number = fields.Char("Building Number")
+    l10n_sa_edi_plot_identification = fields.Char("Plot Identification")
+    l10n_sa_edi_neighborhood = fields.Char("Neighborhood")
+
+    l10n_sa_other_seller_scheme = fields.Selection([
+        ('CRN', 'Commercial Registration Number'),
+        ('MOM', 'Momra License'),
+        ('MLS', 'MLSD License'),
+        ('SAG', 'Sagia License'),
+        ('OTH', 'Other OD')
+    ], default="CRN", string="Identification Scheme", help="Seller Identification scheme")
+
+    l10n_sa_other_seller_id = fields.Char("Seller Identification", copy=False, help="Seller Identification Number")
```

Comment:
I am confused by the term seller, which is normally not the customer.

Reply:
The seller is the Company, not the customer. In this case, Seller Identification is a required field on the seller, and thus the company. I understand the confusion, the field could be moved to res.company instead of res.partner

Reply:
Ok, so better on res.company and without _id as it is a char.  (maybe you can call it _ident or something)

Reply:
I renamed both fields to:

l10n_sa_additional_identification_scheme
l10n_sa_additional_identification_number

They are not just required for sellers, but buyers as well. As such, they have to remain on the res.partner model so we can access them on the customers